wordCountTwitter/twitter-word-count.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <textarea name="" id="textarea-input" cols="30" rows="10"></textarea>
+    <p>Character left : <span id="character-left">0/280</span></p>
```

Comment:
```suggestion
    <p>Character left : <span id="character-left">0</span>/280</p>
```
If you do something like this, then you won't have to append "/280" string in the innerHTML again and again.